docs/source/instruction_set_extensions.rst

```diff
@@ -36,6 +36,12 @@ CV32E40P supports the following CORE-V ISA X Custom Extension, which can be enab
 Additionally the event load instruction (**cv.elw**) is supported by setting ``COREV_CLUSTER`` == 1, see :ref:`corev_event_load`.
 This is a separate ISA extension, invoked in the tool chain with ``-march=rv32i*_xcvelw``.
 
+This specification also includes documentation on CORE-V pseudo-instructions. Pseudo-instructions are implemented in the assembler
+that are similar to a base instruction but provides control informtation to the assembler as opposed to generating its base instruction.
+This makes it easier to program as we gain clarity on the intention of the programmer.
```

Comment:
Pseudo-instructions do not exist in the actual hardware. They are accepted by the assembler and mapped to appropriate actual instructions.  This is very common with standard RISC-V - for example ``nop`` is a pseudo-instruction which maps to `addi x0, x0, 0`.